tests/bdd/steps/feature_steps/fixture.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Fixture helpers
+  Scenario: the user is a {role}
```

Comment:
There is a slight ambiguity here in terms of user ownership: is this a user per the LTI  request or our interpretation of an LTI request and a "user" on the LMS app's end? I believe it is the former, but the ambiguity makes a small confusion here.